_component_design/chips.md

```diff
@@ -4,11 +4,11 @@ parent: Content Areas
 layout: component
 category: Components
 usage: >
-  Chips represent a conceptual unit, such as a user, a device, a filter term, or a chunk of metadata and provide the user with the visual understanding of this conceptual unit as well as the ability to interact with them.
+  Chips represent a block of information, such as a contact, a device, a filter term, or metadata. Chips provide the user with a visual understanding of the information, and the ability to interact with it. Contact information and tags are common types of chips.
```

Comment:
Not sure how I feel about the phrase, "block of information", considering that the word block means a larger amount of information than a unit. For example, I would consider a filter term a "unit of information", and a list of filters terms a, "block on information".  Do chips represent a unit, a block, or both?